src/views/AddItem.jsx

```diff
@@ -19,12 +44,23 @@ export function AddItem({ listToken }) {
 	};
 	const handleSubmit = async (e) => {
 		e.preventDefault();
+		if (!itemName || trimmedItem === '') {
+			setStatus(
+				`You've submitted an empty field. Please enter a valid item name to add to list`,
+			);
```

Comment:
Both this error message and the error message on line 55 ('You already added this item') are not distinct from the success message - it's unclear that this is an error. In terms of UX/UI this could easily lead to a user trying to add the same item again after an error message because they can easily overlook the message. 

Some simple changes would be to: 

- change the text color of these error messages to something distinctly different from black (e.g. red)
- not have the form field clear on submit if there's an error (a cleared form makes users assume the item has been submitted) 

Also when a user tries to add two different items they've already added back-to-back - the lack of modal/pop-up makes it appear as if no new message is being given to the user (the 'You already added this item' message stays constant below the submit button). Having the message animate/pop-up/reload etc will greatly improve the user's experience. I know there was an earlier discussion about the browser error messages vs custom. I think that you can continue to stick to custom as you have already and add these changes.

Here's an example of clear UI around a form field error:
![Screenshot 2023-09-15 at 9 43 37 AM](https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/115492619/72f6f01e-a40d-4426-8834-79d40fbef099)